src/typescript/index.js

```diff
@@ -23,17 +23,21 @@ async function loadTypesIndex() {
   // m = modules associated
   // t = @types/<name>
   body.forEach(type => {
-    typesCache[type.t] = type.t;
+    typesCache[unmangle(type.t)] = type.t;
     type.m.forEach(m => {
-      typesCache[m] = type.t;
+      typesCache[unmangle(m)] = type.t;
     });
   });
 
   datadog.timing('typescript.loadTypesIndex', Date.now() - start);
 }
 
 function isDefinitelyTyped({ name }) {
-  return typesCache[name];
+  return typesCache[unmangle(name)];
+}
+
+function unmangle(name) {
+  return name.replace('__', '/').replace('@', '');
```

Comment:
can we add a test for this? Shouldn't we add an @ instead of removing it?

Reply:
I can not add an @ because the index created by typescript also contain "fake module" like `lodash/add` (I'm not 100% sure I need to add those in the list)

Reply:
Ah that's confusing